bin/start-worker

```diff
@@ -6,6 +6,3 @@ trap 'kill $(jobs -p)' EXIT
 
 # start celery worker with heartbeat (-B)
 SKIP_ASYNC_MIGRATIONS_SETUP=0 celery -A posthog worker -B --scheduler redbeat.RedBeatScheduler --without-heartbeat --without-gossip --without-mingle -Ofair -n node@%h &
-
-# start celery plugin worker
-./bin/plugin-server
```

Comment:
This is going to break some things:
- bin/docker-dev relies on this starting plugin server
- (at least my) local workflows rely on worker starting all worker-related processes

I'd recommend making start-worker start everything plugin-server related as well + document this is for dev, not deployments.